src/components/Table/filters/DateRangeColumnFilter.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, InputLabel, TextField } from '@mui/material'
+import { DesktopDateTimePicker } from '@mui/x-date-pickers/'
+import moment from 'moment'
+import { Fragment, useState } from 'react'
+import { FilterProps } from 'react-table'
+import { TableData } from '../Table'
+import { useActiveElement } from './filterHelpers'
+
+const isValidDateString = (input: string) =>
+  moment.utc(input, 'MM/DD/YYYY HH:mm', true).isValid()
+
+const toUtc = (input: string) => {
```

Comment:
Not sure where to put this comment, but it's UTC related, so I'll put it here.

I'm still kind of confused by the date picker.  It doesn't indicate in the picker itself that the time is UTC and it seems to default to the time in local time.  Meaning if it's 15:15 local time, that's what the default time in the picker will be, but then when I look at the filter it gets applied as 15:15 UTC, which probably wasn't my intention.  The picker should indicate the timezone that you're selecting for and then the starting point should be in that timezone.

Reply:
That's worth pursuing, but it may be rather difficult. As I may have mentioned, I ran into some thorny issues with the date picker. Bad enough that I would have chosen a totally different library, if that was an option (it essentially isn't, as long as this is tied to Material UI). Will go on the TODO list.